redi-connect-front/src/components/organisms/Footer.tsx

```diff
@@ -1,83 +1,101 @@
-            </Columns.Column>
-            <Columns.Column className="is-hidden-tablet footer--headline">
-              <p className="is-size-5">{t('footer.socialMediaHeadline')}</p>
-              <Icons />
+          <Columns breakpoint="mobile">
+            <Columns.Column mobile={{ size: 'full' }}>
+              <Element renderAs="a" href="https://www.redi-school.org" target="_blank">
+                <RediSchool className="oneandhalf-bs" />
+              </Element>
+              <Element renderAs="a" href="https://www.deloitte.de" target="_blank">
+                <Deloitte />
+              </Element>
             </Columns.Column>
-            <Columns.Column className="footer--links">
-              <p>
-                <a
-                  href="https://www.redi-school.org/imprint"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {t('footer.contact')}
-                </a>
-              </p>
-              <p>
-                <a href="/" target="_blank" rel="noopener noreferrer">
-                  {t('footer.faq')}
-                </a>
-              </p>
-              <p>
-                <a
-                  href="https://www.redi-school.org/berlin-transparency/"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {t('footer.transparency')}
-                </a>
-              </p>
-              <p>
-                <a href="/">{t('footer.cookies')}</a>
-              </p>
-              <p>
-                <a
-                  href="https://www.redi-school.org/data-privacy-policy"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {t('footer.dataPolicy')}
-                </a>
-              </p>
+            <Columns.Column mobile={{ offset: 'zero', size: 'half' }} desktop={{ offset: 1 }} className="double-bs">
+              <Element renderAs="ul">
+                <Element renderAs="li" textSize={6} textTransform="uppercase" textWeight="bold">
+                  {t('footer.support')}
+                </Element>
+                <Element renderAs="hr"/>
+                {supportLinks.map(link => (
+                  <Element renderAs="li" textSize={6} textTransform="uppercase">
+                    <Element renderAs="a" href={link.url} target="_blank">
+                      {link.name}
+                    </Element>
+                  </Element>
+                ))}
+              </Element>
             </Columns.Column>
-            <Columns.Column className="is-hidden-mobile is-narrow footer--headline">
-              <p>{t('footer.socialMediaHeadline')}</p>
-              <Icons />
+            <Columns.Column mobile={{ size: 'half' }}>
+              <Element renderAs="ul">
+                <Element renderAs="li" textSize={6} textTransform="uppercase" textWeight="bold">
```

Comment:
Here as well.

Reply:
Headings for Lists have been updated with a more semantic structure